assets/js/src/form_editor/store/mapping/to_blocks/styles_mapper.ts

```diff
@@ -27,22 +27,22 @@ export const mapInputBlockStyles = (styles) => {
     mappedStyles.backgroundColor = styles.background_color;
   }
   if (has(styles, 'border_size') && styles.border_size !== undefined) {
-    mappedStyles.borderSize = parseInt(styles.border_size, 10);
+    mappedStyles.borderSize = +styles.border_size;
```

Comment:
Why this change? I find it less readable and less understandable. I don't understand why is it in this commit, what does it have in common with the rest of the changes in here? Also what happens if `border_size` is an empty string? Can it be?
https://github.com/airbnb/javascript#coercion--numbers